src_python/yaafelib/audiofeature.py

```diff
@@ -25,14 +25,32 @@
 import yaafelib.core as yaafecore
 
```

Comment:
You may need to add `from __future__ import print_function` before any import statements. https://docs.python.org/2/library/2to3.html